src/test/store/actions.test.js

```diff
@@ -140,6 +140,25 @@ describe('selectors/getStackTimingByDepthForStackChart', function() {
   });
 });
 
+describe('selectors/getCallNodeSamplesByDepthForFlameGraph', function() {
```

Comment:
There should of course be a comment here with some nice illustration showing what the flame graph should look like.